ledger-core/v2/insolar/virtual.go

```diff
@@ -165,3 +165,26 @@ type RequestResult interface {
 	Result() []byte
 	ObjectReference() Reference
 }
+
+type ContractCallType uint8
```

Comment:
i guess this type belongs to either virtual or runner, I'm not sure that we need this on cloud scope